src/ontology/edits/oeo-physical.omn

```diff
@@ -598,7 +602,7 @@ Class: OEO_00000025
         rdfs:label "methane"
     
     SubClassOf: 
-        OEO_00000331,
+        OEO_00000437,
```

Comment:
```suggestion
        OEO_00000331,
```
Original classification as portion of matter restored.

Reply:
This change of methane was intentional: Volatile organic compounds are non-methane volatile organic compounds plus methane.

Reply:
That's right and methane *is* also a fuel... But first of all, it is a portion of matter. 

Can we create an equivalent class which puts methane into VOC?

Reply:
So you suggest something like
```
'portion of matter' and ('has disposition' some 'volatile organic compound disposition')
```

Reply:
Exactly

Reply:
What would be a good definition of this new disposition?

_The volatile organic compound disposition is the disposition of a portion of matter to consist of organic compounds with a high vapor pressure at ordinary room temperature._

Reply:
> The volatile organic compound disposition is the disposition of a portion of matter to consist of organic compounds with a high vapor pressure at ordinary room temperature.

I think we should just define _volatility_ as a disposition. Organic compounds are the bearers of the disposition. 

The volatility disposition is the disposition of a portion of matter to sublimate or evaporate from the solid or liquid form and enter the surrounding air. 

Note, ChEBI has 'volatile organic compound': https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:134179

Reply:
Okay, now we have a volatility disposition: f371eb5347ed83712e2ca2c98fe6a8fabce3abb7 

And methane is now a volatile organic compound in the inferred view:
![grafik](https://user-images.githubusercontent.com/36884905/80122314-a72f4f80-858d-11ea-8b41-dafe769c5da2.png)